src/main/java/uk/gov/hmcts/reform/managecase/api/errorhandling/RestExceptionHandler.java

```diff
@@ -41,6 +41,12 @@
         return toResponseEntity(HttpStatus.BAD_REQUEST, ex.getLocalizedMessage());
     }
 
+    @ExceptionHandler(CaseCouldNotBeFetchedException.class)
+    public ResponseEntity<Object> handleCaseCouldNotBeFetchedException(CaseCouldNotBeFetchedException ex) {
+        log.error("Case Id errors: {}", ex.getMessage(), ex);
```

Comment:
`Case Id errors:` seems like a copy-paste error. :)

Reply:
Marking as resolved as code has been updated to:

```log.error("Data Store errors: {}", ex.getMessage(), ex);```

... which seems more appropriate.